modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/widget/WidgetManagerImpl.kt

```diff
@@ -157,7 +162,15 @@ class WidgetManagerImpl @Inject constructor(
             R.id.widget_artwork
         )
         val imageLoader = PodcastImageLoader(context = context, isDarkTheme = true, transformations = emptyList())
-        if (playingEpisode is UserEpisode) {
+        if (playingEpisode is PodcastEpisode) {
+            imageLoader.smallPlaceholder().loadEpisodeArtworkInto(
+                imageView = null,
+                target = target,
+                size = 128,
+                episode = playingEpisode,
+                coroutineScope = this
+            )
```

Comment:
If an episode image is not available for a podcast, then the podcast image should be displayed for the episode. 
I checked loading an episode from `The Daily` on the widget and it could load any image:

https://github.com/Automattic/pocket-casts-android/assets/1405144/7d14c5dc-1216-4747-94d5-cbfcc64e49b3

Reply:
Looks good now